redfish-core/lib/systems.hpp

```diff
@@ -2988,7 +3200,6 @@ inline void requestRoutesSystems(App& app)
  */
 inline void requestRoutesSystemResetActionInfo(App& app)
 {
-
```

Comment:
For next time, empty lines mess up the git blame, make patches tougher to carry downstream